src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +81,17 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ResponseEntity processJSONException(final JSONException e, final NativeWebRequest nativeWebRequest) {
+        if (e.getCause() == null) {
```

Comment:
I do not put stacktrace to the details field. I put only message from JSONException which is json parsing errors in general (https://github.com/stleary/JSON-java).

Also it is not clear how to implement it according to ARUHA-162 because if we show 'Your json request is not array' for every JSONException it will be wrong. We can check for the array our selfs but I think it is more cumbersome.

What do you think ?